class-tour.php

```diff
@@ -50,18 +50,24 @@ public static function enqueue_scripts() {
 		}
 		$tours = array_filter(
 			$_tours,
-			function ( $tour ) {
-				$restrict_url            = $tour[0]['tour_restrict_url'];
-				$create_tour_url_pattern = '#/wp-admin/post\.php\?post=#';
-				if ( empty( $tour[0]['tour_restrict_url'] ) || preg_match( $create_tour_url_pattern, $_SERVER['REQUEST_URI'] ) ) {
+			function ( $tour, $post_id ) {
+				if ( ! isset( $tour[0]['tour_restrict_url'] ) || empty( $tour[0]['tour_restrict_url'] ) ) {
 					return true;
 				}
+				$restrict_url = esc_url( $tour[0]['tour_restrict_url'] );
+
+				$create_tour_url = '/wp-admin/post.php?post=' . esc_html( $post_id ) . '&action=edit';
+				if ( $_SERVER['REQUEST_URI'] === $create_tour_url ) {
```

Comment:
Using `admin_url()` makes it work with installs where the wp-admin is not in `/`.
```suggestion
				$create_tour_url = admin_url( 'post.php?post=' . esc_html( $post_id ) . '&action=edit' );
```

Reply:
in this case we will change
```
if ( $_SERVER['REQUEST_URI'] === $create_tour_url ) {
```
to 
```
if ( strpos( $create_tour_url, $_SERVER['REQUEST_URI'] ) !== false ) {

Reply:
I just know it's good practise to use admin_url. I suppose wp-admin could even be on another domain.